sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryServicesImpl.java

```diff
@@ -424,6 +427,9 @@ public Job getJob(JobReference jobRef, Sleeper sleeper, BackOff backoff)
     private final PipelineOptions options;
     private final long maxRowsPerBatch;
     private final long maxRowBatchSize;
+    // aggregate the total time spent in exponential backoff
```

Comment:
Please consider adding this to Python as well (in  a separate PR).

cc: @pabloem

Reply:
Will do in a new PR.